src/resources/views/crud/inc/column_anchors.blade.php

```diff
@@ -0,0 +1,25 @@
+@php
```

Comment:
Since this is only used inside columns, how do you feel about moving this inside a new ```columns/inc``` folder? Feels like we're going to have multiple views that columns can include - no point in polluting the main ```inc``` folder if it's just specific to columns, right?

Reply:
File structure is likewise important as code structure. It's a good move now that we have 3 files specific to columns.